spec/content_tagger/removing_taxon_spec.rb

```diff
@@ -0,0 +1,37 @@
+feature "Removing content from Content Tagger", new: true, collections: true, content_tagger: true do
```

Comment:
I'd guess this actually isn't coupled to collections?

Reply:
It's being viewed on the collections frontend so we thought it should be linked like the updating one is

Reply:
Sure, should we include that in the expectations then?